hub/powertoys/color-picker.md

```diff
@@ -65,75 +65,75 @@ Color Picker has following settings:
 - Activation shortcut
 - Behavior of activation shortcut
 - The default format of a copied color (HEX, RGB, etc.)
```

Comment:
```suggestion
- **Activation shortcut**
- **Activation behaviour**
- **Default color format**, such as HEX, RGB, and more.
```

I think these should be the exact UI text and, therefore, in bold.

Again, let me know if this is out of scope for this PR!